opentracing_instrumentation/client_hooks/strict_redis.py

```diff
@@ -98,6 +98,8 @@ def execute_command(self, cmd, *args, **kwargs):
 
         # for certain commands we'll add extra attributes such as the redis key
         for tag_key, tag_val in getattr(self, '_extra_tags', []):
+            if isinstance(tag_val, basestring):
+                tag_val = tag_val.encode('string_escape')
```

Comment:
iirc, currently we convert all tag values to strings in the client, but the new data model we're working on will be able to capture original types. We'd still only deal with primitive types, so a binary tag value is fairly useless unless we build some plugins in the UI to be able to decode it for the viz.